dashboards/Insight/Prometheus_Exporter_Status.json

```diff
@@ -105,6 +109,10 @@
   "panels": [
     {
       "collapsed": false,
+      "datasource": {
+        "type": "prometheus",
```

Comment:
ok, uid should be avoided. @anton-bystrov please help with this PR

Reply:
Done